dev-db/sqldeveloper/sqldeveloper-21.2.0.187.1842.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop java-pkg-2
+
+DESCRIPTION="Oracle SQL Developer is a graphical tool for database development"
+HOMEPAGE="https://www.oracle.com/technetwork/developer-tools/sql-developer/overview/index.html"
+SRC_URI="${P}-no-jre.zip"
+
+RESTRICT="fetch"
+
+LICENSE="OTN"
+SLOT="0"
+KEYWORDS="-* ~amd64"
+
+IUSE="mssql mysql postgres sybase"
+
+RDEPEND="
+	>=dev-java/openjdk-8:*[javafx]
+	>=virtual/jre-1.8:*
+	mssql? ( dev-java/jtds:1.3 )
+	mysql? ( dev-java/jdbc-mysql:0 )
+	postgres? ( dev-java/jdbc-postgresql:0 )
+	sybase? ( dev-java/jtds:1.3 )
+"
+BDEPEND="app-arch/unzip"
+
+S="${WORKDIR}/${PN}"
+
+QA_PREBUILT="
+	opt/${PN}/netbeans/platform/modules/lib/amd64/linux/libjnidispatch-422.so
+"
```

Comment:
Don't this need more? Not that it's a big deal given tinderboxes won't be checking a fetch-restricted package.

If a lot of files it's okay to use some wildcards, think only need:
`opt/${PN}/modules/javafx/linux-x64/lib/*.so`

```
 * QA Notice: Files built without respecting CFLAGS have been detected
 *  Please include the following list of files in your report:
 * /opt/sqldeveloper/modules/javafx/linux-x64/lib/libavplugin-54.so
 * /opt/sqldeveloper/modules/javafx/linux-x64/lib/libavplugin-56.so
 * /opt/sqldeveloper/modules/javafx/linux-x64/lib/libavplugin-57.so
 * /opt/sqldeveloper/modules/javafx/linux-x64/lib/libavplugin-ffmpeg-56.so
 * /opt/sqldeveloper/modules/javafx/linux-x64/lib/libavplugin-ffmpeg-57.so
 * /opt/sqldeveloper/modules/javafx/linux-x64/lib/libavplugin-ffmpeg-58.so
 * /opt/sqldeveloper/modules/javafx/linux-x64/lib/libdecora_sse.so
 * /opt/sqldeveloper/modules/javafx/linux-x64/lib/libfxplugins.so
 * /opt/sqldeveloper/modules/javafx/linux-x64/lib/libglass.so
 * /opt/sqldeveloper/modules/javafx/linux-x64/lib/libglassgtk2.so
 * /opt/sqldeveloper/modules/javafx/linux-x64/lib/libglassgtk3.so
 * /opt/sqldeveloper/modules/javafx/linux-x64/lib/libgstreamer-lite.so
 * /opt/sqldeveloper/modules/javafx/linux-x64/lib/libjavafx_font.so
 * /opt/sqldeveloper/modules/javafx/linux-x64/lib/libjavafx_font_freetype.so
 * /opt/sqldeveloper/modules/javafx/linux-x64/lib/libjavafx_font_pango.so
 * /opt/sqldeveloper/modules/javafx/linux-x64/lib/libjavafx_iio.so
 * /opt/sqldeveloper/modules/javafx/linux-x64/lib/libjfxmedia.so
 * /opt/sqldeveloper/modules/javafx/linux-x64/lib/libjfxwebkit.so
 * /opt/sqldeveloper/modules/javafx/linux-x64/lib/libprism_common.so
 * /opt/sqldeveloper/modules/javafx/linux-x64/lib/libprism_es2.so
 * /opt/sqldeveloper/modules/javafx/linux-x64/lib/libprism_sw.so
```

Reply:
Done by now, not related to this problme, I have
```
 * QA Notice: Unresolved soname dependencies:
 * 
 *      /opt/sqldeveloper/modules/javafx/linux-x64/lib/libavplugin-54.so: libavcodec.so.54 libavformat.so.54
 *      /opt/sqldeveloper/modules/javafx/linux-x64/lib/libavplugin-56.so: libavcodec.so.56 libavformat.so.56
 *      /opt/sqldeveloper/modules/javafx/linux-x64/lib/libavplugin-57.so: libavcodec.so.57 libavformat.so.57
 *      /opt/sqldeveloper/modules/javafx/linux-x64/lib/libavplugin-ffmpeg-56.so: libavcodec-ffmpeg.so.56 libavformat-ffmpeg.so.56
 *      /opt/sqldeveloper/modules/javafx/linux-x64/lib/libavplugin-ffmpeg-57.so: libavcodec.so.57 libavformat.so.57
 * 
```
I'm not sure if delete this lib because libavplugin-ffmpeg-58.so is present.

Reply:
Looking closer this seem odd in general. Older ebuilds didn't provide modules/javafx did they? e.g. less qa_prebuilt, no fperms for these libraries. Also seem to be missing a lot of dependencies for this, ffmpeg, opengl, etc..

I also see a dependency on `>=dev-java/openjdk-8:*[javafx]` implying that it's using the system's javafx

Does this work if don't install javafx at all? I tried to delete it and it's working just fine.

Reply:
> Does this work if don't install javafx at all? I tried to delete it and it's working just fine if have right jdk eselect'ed.

Yes seems to work, I tried also datamodeler.
I completely remove javafx.